docs/src/main/sphinx/connector/redshift.md

```diff
@@ -64,6 +64,31 @@ documentation](https://docs.aws.amazon.com/redshift/latest/mgmt/jdbc20-configura
 ```{include} jdbc-authentication.fragment
 ```
 
+### UNLOAD configuration
+
+The connector brings parallelism by converting Trino query to corresponding 
+Redshift `UNLOAD` command to unload query results on to the S3 bucket in the 
+form of Parquet files and reading these Parquet files from S3 bucket. Parquet 
+files will be removed as Trino query finishes. However, it's advisable to 
+define a custom life cycle policy on the S3 bucket used for unloading the 
+Redshift query results.
+This feature intends to bring query results faster compared to the default JDBC 
+approach when fetching large query result set from Redshift.
```

Comment:
```
This feature enables using Amazon S3 to efficiently transfer data out of Redshift
instead of the default single threaded JDBC based implementation.
The connector automatically triggers the appropriate `UNLOAD` command 
on Redshift to extract the output from Redshift to the configured 
S3 bucket in the form of Parquet files. These Parquet files are read in parallel 
from S3 to improve latency of reading from Redshift tables. The Parquet 
files will be removed when Trino finishes executing the query. It is recommended 
to define a custom life cycle policy on the S3 bucket used for unloading the 
Redshift query results.
```